website/docs/docs/deploy/ci-jobs.md

```diff
@@ -11,12 +11,12 @@ You can set up [continuous integration](/docs/deploy/continuous-integration) (CI
 
 dbt Labs recommends that you create your CI job in a dedicated dbt Cloud [deployment environment](/docs/deploy/deploy-environments#create-a-deployment-environment) that's connected to a staging database. Having a separate environment dedicated for CI will provide better isolation between your temporary CI schema builds and your production data builds. Additionally, sometimes teams need their CI jobs to be triggered when a PR is made to a branch other than main. If your team maintains a staging branch as part of your release process, having a separate environment will allow you to set a [custom branch](/faqs/environments/custom-branch-settings) and, accordingly, the CI job in that dedicated environment will be triggered only when PRs are made to the specified custom branch. To learn more, refer to [Get started with CI tests](/guides/set-up-ci).
 
+
 ### Prerequisites
 - You have a dbt Cloud account.
 - For the [Concurrent CI checks](/docs/deploy/continuous-integration#concurrent-ci-checks) and [Smart cancellation of stale builds](/docs/deploy/continuous-integration#smart-cancellation) features, your dbt Cloud account must be on the [Team or Enterprise plan](https://www.getdbt.com/pricing/).
-- You must be connected using dbt Cloud’s native Git integration with [GitHub](/docs/cloud/git/connect-github), [GitLab](/docs/cloud/git/connect-gitlab), or [Azure DevOps](/docs/cloud/git/connect-azure-devops).
```

Comment:
previously, this line implied that the user would need to set up the connection first before proceeding with creating a CI job in the UI. 

with this update, will the user still need to set up the connection first?

Reply:
you technically don't need a repo or a git provider connection to create a CI job, but it will fail as soon as you try to run it. a native git integration is needed to trigger the job on PRs; otherwise, users will see the following message (which is pretty clear IMO)
<img width="1383" alt="Screenshot 2024-01-04 at 11 01 57" src="https://github.com/dbt-labs/docs.getdbt.com/assets/56835998/5766d581-6bc9-42bd-bfb6-f59af18e2727">

Reply:
gotcha!

i'm thinking about how we can document a "happy path" here and your extra context is helpful! that message in the UI is a great guardrail for the user ❇️

Reply:
gotcha. for the "happy path", you would need to connect a repo (either with a native integration or one of the other methods) before running the job. doing so before creating a CI job makes the most sense, but there's nothing stopping you from doing it out of order

Reply:
👍 thank you!

can you take a look at my latest update re: this? i think i captured all the bits this time